subjects/mobile-dev/hacker-news/README.md

```diff
@@ -1,36 +1,31 @@
-# Hacker news
+## Hacker news
 
-### Introduction
+Hacker News is a social news website that mainly focuses on computer science and entrepreneurship. The objective of this task is to create a client with all the main features of the [HackerNewsAPI](https://github.com/HackerNews/API).
 
-Create your own hackernews [app](https://news.ycombinator.com/)!
+### Instructions
 
-Hacker News is a social news website, which mainly focusses on Computer Science and Entrepreneurship. You will make a client consisting of all the main features of the website.
+Your app should have the following functionalities:
 
-### Objective
+- Login button. Once you tap it a log in page must be displayed.
+- Show a list of all fetched `posts` on the main screen.
+- Each `post` should have an `upvote` option for voting.
+- Each `post` should display the total number of votes and comments they have.
+- Display the `username` of the `post` creator.
+- Display the submission time of the `post`.
+- Open the link in a [Webview](https://codelabs.developers.google.com/codelabs/flutter-webview#0) when a `post` is tapped.
```

Comment:
Little detail - in the link `WebView` is the format used for the word

Reply:
done